pkg/gradetool/testdata/out.json

```diff
@@ -0,0 +1,102 @@
+[
+    {
```

Comment:
I mentioned the same thing in the JIRA. Since we are not doing nested grades in the output, should we use an array instead? Ideally, we'd like to keep the grade results in the same order as how they are defined in the policy?

Reply:
Thanks.
What alternative can we suggest?

Reply:
```
[ 
 {
   "name": "good",
   "propose": "true",
   "pass": [],
   "fail": []   
 },
 {
   "name": "better",
   "propose": "true",
   "pass": [],
   "fail": []   
 },
 {
   "name": "best",
   "propose": "false",
   "pass": [],
   "fail": []   
 },
]
```